content/backend/graphql-scala/1-getting-started.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Getting Started
+pageTitle: "Prepare development stack to build GraphQL server"
+description: "In this chapter will be described how to setup the HTTP server, install all dependencies and setup the database."
+question: "Does GraphQL needs HTTP Server?"
+answers: ["Yes. It needs HTTP server.", "Yes, it needs HTTP server but some of features can be used without that", "No, but it strictly recommended to use. Without HTTP layer, GraphQL is losing some of its features.","No, GraphQL is specification is far away from transportation protocol. You can use HTTP, Websockets, sockets or even use it internally in you application." ]
+correctAnswer: 3
+---
+
+In this chapter you will learn how to:
+* Initialize the SBT project,
+* Add all dependencies,
+* Setup basic HTTP server,
+* Setup Database schema and connection.
+
+### Initialize new project
+
+First step will be preparation of SBT project.
+
+I assume you're familiar with any modern IDE which improves your productivity. But, if you don't know any, I recommend to try [IntelliJ IDEA CE](https://www.jetbrains.com/idea/download) which has an awesome plugin for Scala.
+
+Whether you use IDE or not, you have to start from preparing `build.sbt` configuration file.
+
+<Instruction>
+
+Create `build.sbt` file in empty folder with following content:
+
+```scala
+
+name := "howtograph-sangria"
+
+version := "1.0"
+
+description := "GraphQL server with akka-http and sangria"
+
+scalaVersion := "2.12.3"
+
+scalacOptions ++= Seq("-deprecation", "-feature")
+
+libraryDependencies ++= Seq(
+  "org.sangria-graphql" %% "sangria" % "1.3.0",
+  "org.sangria-graphql" %% "sangria-spray-json" % "1.0.0",
+  "com.typesafe.akka" %% "akka-http" % "10.0.10",
+  "com.typesafe.akka" %% "akka-http-spray-json" % "10.0.10",
+
+  "com.typesafe.slick" %% "slick" % "3.2.1",
+  "com.typesafe.slick" %% "slick-hikaricp" % "3.2.1",
+  "org.slf4j" % "slf4j-nop" % "1.6.6",
+  "com.h2database" % "h2" % "1.4.196",
+
+  "org.scalatest" %% "scalatest" % "3.0.4" % Test
+)
+
+Revolver.settings
+
+```
+
+</Instruction>
+
+As you can see at the end of file, I'm using [Revolver Plugin](https://github.com/spray/sbt-revolver). This plugin is really helpful during development. It watches filesystem, recompiles changed files and after that reloads HTTP server. So we are able to see all the changes without restarting the server manually.
+
+Let's add Revolver to the project:
+
+<Instruction>
+
+Create directory `project`. Inside of it create two files: `build.properties` and `plugins.sbt`
+In first place type `sbt.version=0.13.6` to define that we want to use the latest version of SBT.
+
+In the second of these files add dependency to the plugin:
```

Comment:
Now create `plugins.sbt` in `project` and fill it with: {code} This way we will enable revolver plugin faster turnaround times
On top of that, I would drop plugin.sbt from earlier paragraph - IMO it's better to go one after another